cmd/account/get/legal-entity.go

```diff
@@ -101,12 +101,18 @@ func (o *getLegalEntityOptions) run() error {
 				Id:   account.Spec.LegalEntity.ID,
 			}
 
-			outputflag.PrintResponse(o.output, resp)
+			err := outputflag.PrintResponse(o.output, resp)
+			if err != nil {
+				fmt.Println("Error while printing response: ", err.Error())
+			}
 		}
 	}
 
 	// matched account not found
-	fmt.Fprintf(o.IOStreams.Out, "Account matched for AWS Account ID %s not found\n", o.accountID)
+	_, err := fmt.Fprintf(o.IOStreams.Out, "Account matched for AWS Account ID %s not found\n", o.accountID)
```

Comment:
afaik, Fprintf() or Sprintf() does only formatting without spewing the outcome to the stdout
If we omit the first param, would there be a value for using it?

Reply:
There is no value of using the first parameter. It's just an integer stating how much bytes have been written